src/components/Speciality/Events.js

```diff
@@ -1,56 +1,71 @@
 import React from 'react'
-import styled from 'styled-components'
+import { format, isAfter, endOfYesterday } from 'date-fns'
+import { Padding } from 'styled-components-spacing'
+
+import specialityEventIcon from './assets/events-icon.svg'
 import { Row, Col, Grid } from '../grid'
 import { SectionTitle, Subtitle, BodyPrimary } from '../Typography'
 import ExternalAnchor from '../Common/ExternalAnchor'
-import { Padding } from 'styled-components-spacing'
-import { format } from 'date-fns'
+import Hr from '../Common/Hr'
 
-const EventBorder = styled(Col)`
-  border: 1px solid rgba(51, 51, 51, 0.15);
-`
+const noEventsMessage = title =>
+  `It looks like there currently aren’t any upcoming ${title} events. You can always check back again later or get in touch if you are interested in potentially hosting one.`
```

Comment:
is this copy signed off?

Reply:
Yes. See on Zeppelin:
https://app.zeplin.io/project/5b4e666f0d4230a16dffd93a/screen/5cd2acb3eadfea34c03e671e